<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>707</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>PRS20 V1.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ConnectionGroup_Obj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="IP_Label_Obj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Port_Label_Obj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IP_Edit_Obj">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>28</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter IP address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Port_Edit_Obj">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>58</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConButton_Obj">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DisConButton_Obj">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="ConStateLabel_Obj">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TxGroup_Obj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Tx</string>
    </property>
    <widget class="QCheckBox" name="Tx1CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Tx 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Tx2CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Tx 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Tx3CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Tx 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Tx4CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Tx 4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tx1AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tx2AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tx3AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tx4AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx activation and attenuation settings.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RxGroup_Obj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Rx</string>
    </property>
    <widget class="QCheckBox" name="Rx1CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Rx 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Rx2CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Rx 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Rx3CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Rx 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Rx4CheckBox_Obj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Rx 4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rx1AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rx2AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rx3AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rx4AttnEdit_Obj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rx activation and gain settings.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dB</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RcvRawGroup_Obj">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>961</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Received Raw Data</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="ClrScrButton_Obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="TextShow_Obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>941</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LatestFindLabel_Obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>591</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>---&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="SearchBrowser_Obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>941</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton_Obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Search settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FreqGroup_Obj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Frequency</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx and Rx Frequency values.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx L0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rx L0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TxL0FreqEdit_Obj">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>46</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="RxL0FreqEdit_Obj">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>76</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
   </widget>
   <zorder>RcvRawGroup_Obj</zorder>
   <zorder>ConnectionGroup_Obj</zorder>
   <zorder>TxGroup_Obj</zorder>
   <zorder>RxGroup_Obj</zorder>
   <zorder>FreqGroup_Obj</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar_obj"/>
 </widget>
 <resources/>
 <connections/>
</ui>
